### DISCUSSION TITLE
Looking for expert Postal contractor/freelancer

### DISCUSSION THREAD

**Discussion by Digital-Neural** [postalserver/postal#2976] [Help with using Postal]
Hello,
I want to use Postal for an application I'm building. I think it's the ideal platform. I am looking for a contractor/freelancer that can firstly develop on Postal (what we are doing will need some kind of plugin or adaption to work on the mail as it flows through), secondly help set up a production installation of Postal with the potential to scale, thirdly help with the ongoing management of the Postal installation.
If anyone is interested please message me, alternatively if you have any suggestions of where. I should b looking I would really appreciate it.
Thank you

**Comment by vibhoo** [comment]
Hi! I have used postal in the past in the production environment and have helped the client to send out 50K - 1L emails a day. I think I can surely help you out. you can reach out to me at vibhoo[dot]mishra AT gmail.com

**Reply by zhushen12580** [reply]
Hi, how many emails sent by this software will go into the spam of big companies such as Gmail, outlook, etc.?

**Reply by vibhoo** [reply]
Sorry for the late reply. @zhushen12580 It depends on the domain reputation. Need to build up a good domain reputation slowly and steadily, and then the chance of emails going into spam is reduced.

**Comment by Digital-Neural** [comment]
Hopefully none of them.
That being said, we are planning on using this in a very specific
application, not just normal mail handling.
Why do you ask?
